css/demo.css

```
@@ -0,0 +1,194 @@
+@import "resetcss.css";
+@import url('https://fonts.googleapis.com/css?family=Montserrat:300,700|Source+Sans+Pro:300,400');
+
+.container{
+	max-width: 100%;
+	display: flex;
+	display: -webkit-flex;
+	flex-direction: row;
+	background-color: #cad3d3;
+	height: auto;	
+	justify-content: center;
+	align-items: center;
+	flex-wrap: wrap;
+}
+.price-item{
+	width: auto;
+	height: auto;
+	margin: 100px 35px;
+	background-color: #ffffff;
+	text-align: center;
+}
+/*====Thumbnail image====*/
+.price-item-img{
```

nếu nó phụ thuộc vào thằng cha thì nên viết như này em nhé
```suggestion
.price-item .price-item-img{
```
áp dụng luôn cho cái khác